### PR TITLE
[Site-314] Image and blockquote quick fixes on article page

### PR DESCRIPTION
**Pull request by nenispng**
**Link to issue 🔗** 
https://labcodes.atlassian.net/browse/SITE-314

**What's new? ✨** 
- New treatment for images with rounded corners
- Images are now centered and display their original size. If too big for the container, they will fit inside.
- Blockquote simple adjustments. Still doesn't match the redesign, but looks better

**Extra 🌟** 
- New separator style

### CODE REVIEW

File: themes/medius/scss/article.scss [hunk 1]
```diff
@@ -60,6 +60,14 @@
   em {
     font-style: italic;
   }
+
+  hr {
+    height: 1px;
+    border-width: 0;
+    color:$mineral-30;
```
Comment (fernandolins) [inline themes/medius/scss/article.scss:67]: Why do you need this line?

File: themes/medius/scss/article.scss [hunk 2]
```diff
@@ -127,8 +135,19 @@
...
   img {
     display: block;
-    width: 100%;
-    margin: $spacing-5 0;
+    max-width: 100%;
+    border-radius: $baseline;
+    position: relative;
+    margin: auto;
+
+    figcaption {
```
Comment (fernandolins) [inline themes/medius/scss/article.scss:143]: I couldn't test it.
Where can I find a post with a figcaption img?

File: themes/medius/scss/article.scss [hunk 3]
```diff
@@ -261,6 +280,14 @@
       }
     }
   }
+
+  blockquote {
+    border-left: 4px solid $mineral-70;
+    font-style: inherit;
+    margin: 0 0 $spacing-6 0;
+    padding: 0 0 0 $spacing-4;
+    border-radius: $baseline;
```
Comment (fernandolins) [inline themes/medius/scss/article.scss:289]: This way you implemented the left border without rounded edges as we have on Figma UI Design.
What do you think to do it using `:before`?

**Review by fernandolins — CHANGES_REQUESTED** [review]

File: themes/medius/scss/article.scss (hunk 1 above)
Comment (nenispng) [reply themes/medius/scss/article.scss:67]: this is the separator line

File: themes/medius/scss/article.scss (hunk 2 above)
Comment (nenispng) [reply themes/medius/scss/article.scss:143]: we currently dont have any, but we should. the correct way to insert image capitions is using <figcaption>

File: themes/medius/scss/article.scss (hunk 3 above)
Comment (nenispng) [reply themes/medius/scss/article.scss:289]: I tried but I didnt acheive any good results. We can pair later so you can help me.

File: themes/medius/scss/article.scss (hunk 1 above)
Comment (fernandolins) [reply themes/medius/scss/article.scss:67]: I know.
but you can remove the line 67.
add color is not necessary.

File: themes/medius/scss/article.scss (hunk 3 above)
Comment (fernandolins) [reply themes/medius/scss/article.scss:282]: Sure, we can do it later.

File: themes/medius/scss/article.scss (hunk 2 above)
Comment (fernandolins) [reply themes/medius/scss/article.scss:140]: ok, but how can I test it?

Comment (nenispng) [reply themes/medius/scss/article.scss:140]: we can't, yet. we need to inform people to use a figcaption in their next posts.

File: themes/medius/scss/article.scss (hunk 1 above)
Comment (nenispng) [reply themes/medius/scss/article.scss:67]: why not? this is the color I want to apply.

Comment (nenispng) [reply themes/medius/scss/article.scss:67]: sorry. now I see the mistake :neckbeard:

File: themes/medius/scss/article.scss (hunk 2 above)
Comment (fernandolins) [reply themes/medius/scss/article.scss:140]: ok. But I want to test it.
how can I use figcaption locally?

Comment (nenispng) [reply themes/medius/scss/article.scss]: @fernandolins I have chosen to remove the <figcaption> style so we can make it simpler and move on with this PR.

File: themes/medius/scss/article.scss [hunk 4]
```diff
@@ -331,6 +348,11 @@
         }
       }
     }
+
+    blockquote {
+      padding: $baseline 0 $baseline $spacing-5;
```
Comment (fernandolins) [inline themes/medius/scss/article.scss:353]:
```suggestion
      padding: $baseline 0 $baseline $spacing-6;
```

File: themes/medius/scss/article.scss [hunk 5]
```diff
@@ -254,6 +263,14 @@
...
     }
   }
+
+  blockquote {
+    border-left: 4px solid $mineral-70;
+    font-style: inherit;
+    margin: 0 0 $spacing-6 0;
+    padding: 0 0 0 $spacing-4;
+    border-radius: $baseline;
+  }
```
Comment (fernandolins) [inline themes/medius/scss/article.scss:273]:
```suggestion
  blockquote {
    &:before {
      content: "";
      position: absolute;
      left: 0;
      top: 0;
      height: 100%;
      border: 2px solid $mineral-70;
      border-radius: $baseline;
    }

    font-style: inherit;
    margin: 0 0 $spacing-6 0;
    padding: 0 0 0 $spacing-5;
    border: none;
    position: relative;
  }
```

**Review by fernandolins — CHANGES_REQUESTED** [review]

**Review by fernandolins — APPROVED** [review]
LGTM

**Pull request merged**